wsi/physicaldisplay.h

```diff
@@ -116,8 +117,11 @@ class PhysicalDisplay : public NativeDisplay, public DisplayPlaneHandler {
   /**
   * API for setting color correction for display.
   */
-  virtual void SetColorCorrection(struct gamma_colors gamma, uint32_t contrast,
-                                  uint32_t brightness) const = 0;
+  virtual void SetColorCorrection(struct gamma_colors gamma,
+                                  uint32_t contrast,
```

Comment:
hmm this will break Linux tests. I think we need to adapt Linux tests to use the new API and that way also test this actually

Reply:
Did not find SetColorCorrection used in Linux tests

Reply:
hmm ya I thought we had some tests

Reply:
@gaochun , public/nativedisplay.h, SetGamma(), SetContrast(), SetBrightness() are the APIs of Linux side, tests/jsonconfigs/colorcorrection.json is the test json. 

about SetColorCorrection(), ./wsi/drm/drmdisplay.cpp already had, seems some different with yours, let me confirm with you about details offline.

Reply:
@yugang  @kalyankondapally 
What is the problem about this PR
We need to move forward